frontend/packages/schema-editor/src/components/NodePanel/NodePanel.tsx

```diff
@@ -21,11 +21,11 @@ export const NodePanel = ({ schemaPointer }: NodePanelProps) => {
 
   return (
     <>
-      <div className={classes.top}>
-        {!isDataModelRoot && <BackButton />}
-        <HeadingRow schemaPointer={schemaPointer} />
+      <HeadingRow schemaPointer={schemaPointer} />
+      {!isDataModelRoot && <BackButton />}
```

Comment:
Jeg er usikker på om vi burde plassere `<BackButton>` under overskriften. Blir det semantisk riktig? Dette er jo ikke innhold som er spesifikt for akkurat denne overskriften.